### PR TITLE
docs: update headless CI tutorial

### PR DESCRIPTION
**Pull request by erickzhao**
#### Description of Change

Updates information in the Headless Testing doc. This was prompted by some website feedback that the CircleCI link was broken.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/testing-on-headless-ci.md [hunk 1]
```diff
@@ -47,14 +48,15 @@ install:
...
 
-For Jenkins, a [Xvfb plugin is available](https://wiki.jenkins-ci.org/display/JENKINS/Xvfb+Plugin).
+For Jenkins, an [Xvfb plugin is available](https://plugins.jenkins.io/xvfb/).
 
 ### Circle CI
 
-Circle CI is awesome and has Xvfb and `$DISPLAY`
-[already set up, so no further configuration is required](https://circleci.com/docs/environment#browsers).
+Circle CI has Xvfb and the `$DISPLAY` variable
+[already set up](https://circleci.com/docs/2.0/browser-testing/), so no
```
Comment (jkleinsc) [inline docs/tutorial/testing-on-headless-ci.md:56]: This link doesn't explicitly say that xvfb is setup and this link seems to indicate only certain legacy images have xvfb installed: https://circleci.com/docs/2.0/circleci-images/#pre-installed-tools

**Review by jkleinsc — CHANGES_REQUESTED** [review]
The CircleCI section doesn't seem to be accurate to me.

File: docs/tutorial/testing-on-headless-ci.md (hunk 1 above)
Comment (erickzhao) [reply docs/tutorial/testing-on-headless-ci.md:56]: Very good catch. Sorry for the careless addition -- I'll try to do some more research on this.

**Comment by miniak** [comment]
Agreed with @erickzhao to close this for now. He might revive this effort later on.

**Pull request closed without merging**